.gitattributes

```diff
@@ -0,0 +1 @@
+tests/data/storage.json filter=lfs diff=lfs merge=lfs -text
```

Comment:
I'm not sure lfs is a good way to store this.
If we get multiple unrelated data stored like this, your stress test will have to `git lfs init` the whole repo before it can access its blob.

would it be possible to store this on a S3 bucket instead?

Reply:
Yeah makes sense, can put it on S3